static/js/main.js

```diff
@@ -1,33 +1,59 @@
 "use strict";
 
-(function()
-{
-  function initLanguageSelection()
-  {
-    var locale = document.getElementById("navbar-locale-selected");
-
-    // skip if page does not have language selection (EG: blog)
-    if (!locale) return;
-
-    locale.addEventListener("click", function()
-    {
-      document.getElementById("navbar-locale-menu")
-        .classList.toggle("visible");
-    }, false);
-  }
-
-  function navigationClick(event)
-  {
-    document.getElementById("navbar-menu")
-      .classList.toggle("visible");
-  }
-
-  function initMenu()
-  {
-    document.getElementById("navbar-menu-toggle")
-      .addEventListener("click", navigationClick, false);
-  }
-
-  initLanguageSelection();
-  initMenu();
+(function() {
+    function initLanguageSelection() {
+        var locale = document.getElementById("navbar-locale-selected");
+
+        // skip if page does not have language selection (EG: blog)
+        if (!locale) return;
+
+        locale.addEventListener("click", function() {
+            document.getElementById("navbar-locale-menu")
+                .classList.toggle("visible");
+        }, false);
+    }
+
+    function navigationClick(event) {
+        document.getElementById("navbar-menu")
+            .classList.toggle("visible");
+    }
+
+    function initMenu() {
+        document.getElementById("navbar-menu-toggle")
+            .addEventListener("click", navigationClick, false);
+    }
+
+    function navbarToggle() {
+        var navBar = document.getElementById("navbar");
+        var navbarHeight = navBar.offsetHeight;
+        var scrollHandled = false;
+        var lastScrollTop = 0;
+
+        window.addEventListener("scroll", (function() {
+            scrollHandled = true;
```

Comment:
NIT: "handled" usually means "taken care of". AFAICT (As Far As I Can Tell) the purpose of setting this boolean in this function is to flag something "to be handled" and not "as taken care of". Therefore, setting "scrollHandled" (which I interpret as "has the scroll been handled"?) to "true" ("yes!") is misleading IMO.

I suggest setting it to false here and checking `!scrollHandled` below instead of `scrollHandled` and returning `true` from `handleScroll` instead of `false`.